examples/ocaml_parser/driver.ml

```diff
@@ -181,19 +181,20 @@ module Printer = Merlin_recovery.DummyPrinter (I)
 (*    end) *)
  
 module R =
-  Merlin_recovery.Make(I)
+  MenhirRecoveryLib.Make(I)
     (struct
       include Parser_recover
 
-      let default_value loc x =
-        Default.default_loc := loc;
-        default_value x
+(* FIXME *)
+      (* let default_value loc x = *)
+      (*   Default.default_loc := loc; *)
+      (*   default_value x *)
```

Comment:
There are some FIXMEs in this code that perhaps we should explain or address before merging.